zkEVM docs review (#485)
0xPolygon/polygon-docs: docs/zkEVM/architecture/protocol/zkevm-bridge/exit-tree.md

> The zkEVM's bridge SC utilizes a special Merkle tree called Exit Tree for each of the networks participating in the communication or asset-exchange.

Reviewer: ```suggestion
The zkEVM's bridge SC utilizes a special Merkle tree called an exit tree for each of the networks participating in the communication or asset-exchange.
```

> The term `exit tree refers to an append-only Sparse Merkle tree (SMT) whose leaf nodes record information about assets being transferred out of the network. Exit trees of depth 32 are used by Polygon zkEVM.

Reviewer: ```suggestion
The term exit tree refers to an append-only sparse Merkle tree (SMT) whose leaf nodes record information about assets being transferred out of the network. Exit trees of depth 32 are used by Polygon zkEVM.
```

From now on, a leaf of an exit tree is referred to as a exit leaf. Exit leaves are classified into two types: type 0 for recording asset(s) information and Type 1 for recording messaging information.

An exit leaf, in particular, is a Keccak256 hash of the ABI encoded packed structure with the following parameters:

- uint8 leafType: [0] asset, [1] message.
- int32 originNetwork: Origin network ID, where the original asset belongs.
- address originAddress: If `leafType = 0`, Origin network token address (`0x0000...0000`) is reserved for ether. If `leafType = 1`, `msg.sender` of the message.
- uint32 destinationNetwork: Bridging destination network ID.
- address destinationAddress: Address that receives the bridged asset in the destination network.
- uint256 amount: Amount of tokens/ether to bridge.
- bytes32 metadataHash: Hash of the metadata. This metadata contains information about asset transferred or the message payload.

> When a user commits to transferring assets from one network to another, the bridge SC must add an exit leaf to that network's exit tree.

Reviewer: ```suggestion
When a user commits to transferring assets from one network to another, the bridge contract adds an exit leaf to that network's exit tree.
```

Metadata: 0x0...
```

With the newly added exit leaf, the L1 exit tree now has a new root. Updating the global exit tree with this new L1 exit tree root means changing the root of the global exit tree.

To claim the bridged assets on the destination L2 network, the global exit root is verified using a Merkle proof. This proof allows one to confirm whether an exit leaf, containing information about assets being bridged to L2, is represented in the global exit tree by the corresponding L1 exit tree root.

![Updating L1 exit tree and the global exit root](../../../../img/zkEVM/03pzb-exit-leaf-add-L1-L2.png)

### Transfer from rollup L2 to L1

> Transfers can also occur from an L2 Rollup to the Mainnet L1. In this scenario, the same procedure outlined in the previous example is followed, but in the reverse direction.

Reviewer: ```suggestion
Transfers can also occur from an L2 Rollup to the L1 mainnet. In this scenario, the same procedure outlined in the previous example is followed, but in the reverse direction.
```
